src/test-run/browser-dialogs.js

```diff
@@ -0,0 +1,25 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+export const TYPE = {
+    beforeUnload: 'beforeunload',
+    alert:        'alert',
+    confirm:      'confirm',
+    prompt:       'prompt'
+};
+
+export class ExpectedDialog {
+    constructor (type, result) {
+        this.type   = type;
+        this.result = result;
```

Comment:
It's what we need to return from handler function.
https://github.com/DevExpress/testcafe/pull/621/files#diff-6e63fd8c94cb30d26fd63d78cf8d0342R73